blockchain/accept.go

```diff
@@ -217,7 +232,6 @@ func (b *BlockChain) maybeAcceptBlock(block *dcrutil.Block,
 	// if this is the genesis block.
 	prevNode, err := b.getPrevNodeFromBlock(block)
 	if err != nil {
-		log.Debugf("getPrevNodeFromBlock: %v", err)
```

Comment:
That's a mistake on my end.  vim adds "gopkg.in/acd.v0/internal/log" with that line there.  I deleted it and forgot to add it back and save it not using vim.

Pain in the buttocks...